app/controllers/katello/api/v2/host_collections_controller.rb

```diff
@@ -151,6 +152,12 @@ def copy
 
     private
 
+    def find_max_content_host_count
+      if !params['unlimited_content_hosts'] && params['max_content_hosts'].nil?
+        fail HttpErrors::BadRequest, _("unlimited_content_hosts is false but there is no value for max_content_hosts")
+      end
+    end
```

Comment:
I realized after creating this PR that this logic wasn't quite correct. I saw that I broke a test. Better logic incoming!